<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1338</width>
    <height>919</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Naselje</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Tip upor.</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>70</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Vsi uporabniki</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Privat stranka</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Kotlovnice</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Večstanovanjski objekti</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>80</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Vkjuči neaktivne</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbIzpisi">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>90</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Izpiši na ekran</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbZacniZnova">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>30</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Začni znova</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbPrikazi">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>60</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Dodaj ulico na seznam</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbObcina">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QComboBox" name="cbNaselje">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cbUlica">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Občina</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Ulica</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="twPlaniranje">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>1241</width>
      <height>651</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>10</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Izbrane ulice</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="tbSeznamUlic">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>31</y>
      <width>221</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cbZaposleni">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Dimikar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Pričakovana realizacija:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="planiranjeLeReal">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>169</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbIzpisiExcel">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>160</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Izpiši tabelov EXCEL</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="planiranjeHsOd">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>10</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="planiranjeHsDo">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>10</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1390</x>
      <y>830</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ulica</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>HŠ od</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>10</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>do</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1338</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
